packages/imperative/src/plugins/cmd/list/list.handler.ts

```diff
@@ -48,21 +48,25 @@ export default class ListHandler implements ICommandHandler {
         let listOutput: string = "";
         let firstTime = true;
 
-        for (const pluginName in installedPlugins) {
+        for (const pluginName of Object.keys(installedPlugins).sort((a, b) => a.localeCompare(b))) {
             if (Object.prototype.hasOwnProperty.call(installedPlugins, pluginName)) {
                 // Build the console output
-                if (firstTime) {
-                    listOutput = `\n${chalk.yellow.bold("Installed plugins:")} \n\n`;
-                }
+                if (!params.arguments.short) {
+                    if (firstTime) {
+                        listOutput = `\n${chalk.yellow.bold("Installed plugins:")} \n\n`;
+                    }
 
-                listOutput = listOutput + `${chalk.yellow.bold(" -- pluginName: ")}` +
-                  `${chalk.red.bold(pluginName)} \n`;
-                listOutput = listOutput + `${chalk.yellow.bold(" -- package: ")}` +
-                  `${chalk.red.bold(installedPlugins[pluginName].package)} \n`;
-                listOutput = listOutput + `${chalk.yellow.bold(" -- version: ")}` +
-                  `${chalk.red.bold(installedPlugins[pluginName].version)} \n`;
-                listOutput = listOutput + `${chalk.yellow.bold(" -- registry: ")}` +
-                  installedPlugins[pluginName].registry + "\n\n";
+                    listOutput = listOutput + `${chalk.yellow.bold(" -- pluginName: ")}` +
+                        `${chalk.red.bold(pluginName)} \n`;
+                    listOutput = listOutput + `${chalk.yellow.bold(" -- package: ")}` +
+                        `${chalk.red.bold(installedPlugins[pluginName].package)} \n`;
+                    listOutput = listOutput + `${chalk.yellow.bold(" -- version: ")}` +
+                        `${chalk.red.bold(installedPlugins[pluginName].version)} \n`;
+                    listOutput = listOutput + `${chalk.yellow.bold(" -- registry: ")}` +
+                        installedPlugins[pluginName].registry + "\n\n";
+                } else {
+                    listOutput += `${chalk.yellow(pluginName)}@${installedPlugins[pluginName].version}\n`;
```

Comment:
Not sure whether to show `pluginName@version` or `package@version`? Can't remember seeing a case where `pluginName` and `package` are different.

Reply:
Perhaps they are always the same. If they are ever different, a user who does a 'plugin list' is to get the correct name to be able to run 'plugin uninstall' would probably need the package name. Can you think of other use cases in which the pluginName would be the needed output?

Reply:
It looks like the `plugins uninstall` and `plugins update` commands assume their positional argument is present as a key in the plugins JSON file. So using `pluginName` in this case which is also a key in the plugins JSON file makes sense.